kratos/solving_strategies/schemes/residualbased_incremental_aitken_static_scheme.h

```diff
@@ -190,7 +193,7 @@ class ResidualBasedIncrementalAitkenStaticScheme : public ResidualBasedIncrement
         }
 
         // Store results for next iteration
-        noalias(mPreviousDx) = Dx;
+        boost::numeric::ublas::noalias(mPreviousDx) = Dx;
```

Comment:
why should u do this? noalias is both in ublas and in amatrix...so to have it portable we should not add the namespace here...

Reply:
also, it would be best to use the function of the space for doing this

Reply:
Because both Kratos and ublas are defining this function with templates and call is ambiguous